tools/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Azure SDK Tools Contribution Guidelines
+
+This is the repository of tool libraries used by Azure SDK purpose.
```

Comment:
```suggestion
This is the repository used to host tools and libraries used by Azure SDK team engineering system. 
```

Reply:
This is the repository used to host tools and libraries used by _the_ Azure SDK team engineering system.